go.mod

```diff
@@ -129,11 +129,11 @@ require (
 	go.uber.org/zap v1.24.0 // indirect
 	golang.org/x/exp v0.0.0-20230321023759-10a507213a29 // indirect
 	golang.org/x/mod v0.10.0 // indirect
-	golang.org/x/net v0.8.0 // indirect
-	golang.org/x/sync v0.1.0 // indirect
-	golang.org/x/text v0.8.0 // indirect
+	golang.org/x/net v0.10.0 // indirect
+	golang.org/x/sync v0.2.0 // indirect
+	golang.org/x/text v0.9.0 // indirect
 	golang.org/x/time v0.0.0-20220922220347-f3bd1da661af // indirect
-	golang.org/x/tools v0.7.0 // indirect
```

Comment:
I wasn't expecting any changes to `go.mod` or `go.sum`. Do you know why these are here?

Also the change to `.vscode/settings.json` seems unnecessary. Shall we snip that off this PR? We can always raise a new one if we feel strongly about that change.

Reply:
I will revert both of those changes. I ran a `go mod tidy` command that automatically updated the `go.mod` and `go.sum` files